plat/intel/soc/stratix10/include/s10_mailbox.h

```diff
@@ -0,0 +1,119 @@
+#ifndef __S10_MBOX__
+#define __S10_MBOX__
+
+#define MBOX_OFFSET		0xffa30000
+
+#define MBOX_ATF_CLIENT_ID	0x1
+#define MBOX_JOB_ID		0x1
+
+/* Mailbox interrupt flags and masks */
+#define MBOX_INT_FLAG_COE	0x1
+#define MBOX_INT_FLAG_RIE	0x2
+#define MBOX_INT_FLAG_UAE	0x100
+#define MBOX_COE_BIT(INTERRUPT)	(INTERRUPT & 0x3)
```

Comment:
Guard macro arguments inside (). There are a few occurrences here between lines 13 and 29.